bitex/api/REST/rest.py

```diff
@@ -111,4 +116,8 @@ def public_query(self, method_verb, endpoint, **request_kwargs):
         :return: request.Response() object
         """
         request_kwargs['url'] = self.generate_url(self.generate_uri(endpoint))
+        if 'endpointwithversion' in request_kwargs:
+            if request_kwargs['endpointwithversion']:
+                request_kwargs['url'] = endpoint
+                del request_kwargs['endpointwithversion']
```

Comment:
Hm. I'm not sold on this. `RESTAPI.generate_uri()` already creates a version+endpoint string. What's reason for having this keyword?

Reply:
still not clear how to use api version. generate_uri just use self.version, so how to call v1 api when class init as v2? for example, use bitfinex v2, but only v1 has symbol list